modules/builds-create-imagestreamtag.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//* assembly/builds
+//* builds/running-entitled-builds.adoc
+
+[id="builds-create-imagestreamtag-{context}"]
+= Creating an imagestreamtag to the Red Hat Container Catalog
+
+To run entitled builds, you must create an `imagestreamtag` to
+`registry.access.redhat.io/rhel7/rhel:latest`.
+
+.Procedure
```

Comment:
@adambkaplan would you mind helping me update this section, please? I pulled content from our old API guide to get this in place, but I know the procedure is wrong. Thanks!